config/env/default.js

```diff
@@ -10,5 +10,7 @@ module.exports = {
 	port: process.env.PORT || 3000,
 	templateEngine: 'swig',
 	sessionSecret: 'MEAN',
-	sessionCollection: 'sessions'
+	sessionCollection: 'sessions',
+	logo: 'modules/core/img/brand/logo.png',
+	favicon: '/modules/core/img/brand/favicon.ico'
```

Comment:
@codydaig I know you just moved the value, but I think the project has been following mainly relative paths.  Can you change to relative and test?